<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CP Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="ui">
      <item>
       <layout class="QVBoxLayout" name="left_pane" stretch="0,0">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="compile">
            <property name="text">
             <string>Compile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runOnly">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="run">
            <property name="text">
             <string>Compile and Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_pane">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="compiler_out">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Compiler Messages and stderr</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="compiler_edit">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="input_text">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Input or stdin</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="input">
            <item>
             <widget class="QPlainTextEdit" name="in1">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="in2">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="in3">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="output_text">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Output (stdout and stderr)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="output">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPlainTextEdit" name="out1">
                <property name="contextMenuPolicy">
                 <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="out1_verdict">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Verdict : Accepted</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="out1_diff">
                  <property name="text">
                   <string>More</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPlainTextEdit" name="out2">
                <property name="contextMenuPolicy">
                 <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="out2_verdict">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Verdict : Accepted</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="out2_diff">
                  <property name="text">
                   <string>More</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPlainTextEdit" name="out3">
                <property name="contextMenuPolicy">
                 <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="out3_verdict">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Verdict : Wrong Answer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="out3_diff">
                  <property name="text">
                   <string>More</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionAuto_Save"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Misc</string>
    </property>
    <addaction name="actionBeta_Updates"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionReset_Settings"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="onlyRun"/>
    <addaction name="actionRun"/>
    <addaction name="actionDetached_Execution"/>
    <addaction name="actionFormat"/>
    <addaction name="actionKill_Processes"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChange_compile_command"/>
    <addaction name="actionSet_Code_Template"/>
    <addaction name="actionChange_format_command"/>
    <addaction name="actionRun_Command"/>
    <addaction name="actionChange_run_command"/>
    <addaction name="actionSet_Tab_Size"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <widget class="QMenu" name="menuLanguage_2">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionC_C"/>
     <addaction name="actionJava"/>
     <addaction name="actionPython"/>
    </widget>
    <addaction name="actionAuto_Indentation"/>
    <addaction name="actionAuto_Parenthesis"/>
    <addaction name="actionDark_Theme"/>
    <addaction name="actionUse_Tabs"/>
    <addaction name="actionWrap_Text"/>
    <addaction name="actionFont"/>
    <addaction name="menuLanguage_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditor"/>
   <addaction name="menuActions"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="whatsThis">
    <string>Opens a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save file</string>
   </property>
   <property name="whatsThis">
    <string>Saves the file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="statusTip">
    <string>Save as new</string>
   </property>
   <property name="whatsThis">
    <string>Save as a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Compile and Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>Format</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="actionChange_compile_command">
   <property name="text">
    <string>Compile Command</string>
   </property>
  </action>
  <action name="actionChange_run_command">
   <property name="text">
    <string>Runtime Argument</string>
   </property>
  </action>
  <action name="actionChange_format_command">
   <property name="text">
    <string>Format command</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="whatsThis">
    <string>Close the editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionWrap_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wrap Text</string>
   </property>
  </action>
  <action name="actionAuto_Indentation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Indentation</string>
   </property>
  </action>
  <action name="actionAuto_Parenthesis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Parenthesis</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionReset_Settings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
  </action>
  <action name="onlyRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSet_Code_Template">
   <property name="text">
    <string>Choose Template</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Java</string>
   </property>
  </action>
  <action name="actionC_C">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C/C++</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionRun_Command">
   <property name="text">
    <string>Run Command</string>
   </property>
  </action>
  <action name="actionKill_Processes">
   <property name="text">
    <string>Kill Processes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Save</string>
   </property>
  </action>
  <action name="actionBeta_Updates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Beta</string>
   </property>
  </action>
  <action name="actionDetached_Execution">
   <property name="text">
    <string>Detached Execution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionUse_Tabs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Tabs</string>
   </property>
  </action>
  <action name="actionSet_Tab_Size">
   <property name="text">
    <string>Set Tab Size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
